src/it/java/io/weaviate/integration/SearchITest.java

```diff
@@ -0,0 +1,227 @@
+package io.weaviate.integration;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.InstanceOfAssertFactories;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+
+import io.weaviate.ConcurrentTest;
+import io.weaviate.client6.WeaviateClient;
+import io.weaviate.client6.v1.collections.Property;
+import io.weaviate.client6.v1.collections.Reference;
+import io.weaviate.client6.v1.collections.VectorIndex;
+import io.weaviate.client6.v1.collections.VectorIndex.IndexingStrategy;
+import io.weaviate.client6.v1.collections.Vectorizer;
+import io.weaviate.client6.v1.collections.object.Vectors;
+import io.weaviate.client6.v1.collections.query.GroupedQueryResult;
+import io.weaviate.client6.v1.collections.query.MetadataField;
+import io.weaviate.client6.v1.collections.query.NearText;
+import io.weaviate.client6.v1.collections.query.NearVector;
+import io.weaviate.containers.Container;
+import io.weaviate.containers.Container.ContainerGroup;
+import io.weaviate.containers.Contextionary;
+import io.weaviate.containers.Weaviate;
+
+public class SearchITest extends ConcurrentTest {
+  private static final ContainerGroup compose = Container.compose(
+      Weaviate.custom()
+          .withContextionaryUrl(Contextionary.URL)
+          // .withClipInferenceApi(Multi2VecClip.URL)
+          .build(),
+      // Container.MULTI2VEC_CLIP, // Uncomment for testNearImage
```

Comment:
Can these commented-out lines be removed?

Reply:
This is related to the `@Ignored` test below, which I couldn't make work just yet.
I'll try with using the same model that is used to test the Python client and, if that doesn't work, I'll remove the test and this setup altogether 👍

Reply:
I switched to `img2vec-neural` vectorizer for this integration test in https://github.com/weaviate/java-client/pull/381/commits/1e3c531d9bc38057bc8b012caf34840a748b8c8c